sdk/tests/tutorials/marketdata/test_corporate_actions.py

```diff
@@ -1,49 +1,228 @@
 import unittest
+import uuid
+import pytz
+from datetime import datetime, timedelta
 
 import lusid
 import lusid.models as models
 from lusidfeature import lusid_feature
 from lusid.utilities.api_client_builder import ApiClientBuilder
 from utilities.credentials_source import CredentialsSource
 from utilities.test_data_utilities import TestDataUtilities
-import uuid
 
 
 class CorporateActions(unittest.TestCase):
-
     @classmethod
     def setUp(cls):
         # create a configured API client
-        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+        api_client = ApiClientBuilder().build(str(CredentialsSource.secrets_path()))
```

Comment:
This conversion was happening implicitly but I've made it explicit so it's clear. Iirc, `.build` expects a string, but `secrets_path` returns a `Path` object.